TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Robot.java

```diff
@@ -2,18 +2,20 @@
 
 import com.qualcomm.robotcore.eventloop.opmode.OpMode;
 import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
-import com.qualcomm.robotcore.hardware.HardwareMap;
 
 @TeleOp
 public class Robot extends OpMode {
 
+    Drivetrain drivetrain = new Drivetrain();
+
+
     @Override
     public void init() {
-
+        drivetrain.init();
     }
 
     @Override
     public void loop() {
-
+        drivetrain.loop();
```

Comment:
This is good, however the drivetrain cant move because you arent giving the robot the inputs it needs to function. Add a gamepad / controller to have it actually move

Reply:
I forgot to add this, but change the name of the method to something more readable like .drive or .mechanum drive, as loop is to generic and could mean a ton of different things